recipe/meta.yaml

```diff
@@ -10,9 +10,16 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    ################################################################
+    # Apply patch to skip some tests if C-extensions not built.    #
+    #                                                              #
+    # xref: https://github.com/zarr-developers/numcodecs/pull/247  #
+    ################################################################
+    - patches/PR_247.patch
```

Comment:
Looks like we are running some C-extension dependent tests on PyPy even though those C-extensions don't exist and are not available for `import`. Submitted PR ( https://github.com/zarr-developers/numcodecs/pull/247 ) upstream to skip those tests if the associated C-extension is not built. Have applied it here temporarily to bypass this issue.